crates/ty_python_semantic/resources/mdtest/annotations/invalid.md

```diff
@@ -95,7 +95,11 @@ async def outer():  # avoid unrelated syntax errors on yield, yield from, and aw
 
 ## Invalid Collection based AST nodes
 
+<!-- snapshot-diagnostics -->
```

Comment:
Enabling snapshots for this subsection creates a snapshot that's sort-of overwhelmingly huge 😆 I'm inclined to add a new subsection to the `## Diagnostics for common errors` section below, enable snapshots for that new subsection, and add duplicate tests to that section for the nodes where we offer tailored hints. It doesn't matter too much if some nodes are tested twice, once without snapshots and once with snapshots!